modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,715 @@
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-describe[secrets-mgr:tls-context:mule:describe]| Shows details of a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-list[secrets-mgr:tls-context:mule:list]| Lists all Mule TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-modify[secrets-mgr:tls-context:mule:modify]| Modifies a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-replace[secrets-mgr:tls-context:mule:replace]| Replaces an existing Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-create[secrets-mgr:tls-context:flex-gateway:create]| Creates a new Flex Gateway TLS Context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-describe[secrets-mgr:tls-context:flex-gateway:describe]| Shows details of a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-list[secrets-mgr:tls-context:flex-gateway:list]| Lists all Flex Gateway TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-modify[secrets-mgr:tls-context:flex-gateway:modify]| Modifies a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-replace[secrets-mgr:tls-context:flex-gatway:replace]| Replaces an existing Flex Gateway TLS context secret
+
+
+|===
+
+// end::summary[]
+
+// tag::commands[]
+
+[[secret-group-create]]
+== secrets-mgr:secret-group:create
+
+----
+> secrets-mgr:secret-group:create [flags] 
+----
+
+Creates a new secret group with the name specified in `--name`.
+
+Prompt the `--downloadable` flag if the secrets in this group are referenced in an API Manager proxy.
+
+This command also accepts the default `--help` flag.
+
+
+[[secret-group-delete]]
+== secrets-mgr:secret-group:delete
+
+----
+> secrets-mgr:secret-group:delete [flags] 
+----
+Deletes the secret group specified in `--id` +
+
+[WARNING]
+This command does not prompt twice before deleting. If you send a delete instruction, it does not ask for confirmation.
+
+This command accepts the default flag `--help`.
+
+[[secret-group-describe]]
+== secrets-mgr:secret-group:describe
+
+----
+> secrets-mgr:secret-group:describe [flags]
+----
+
+Returns the details of a secret group specified in `--id`. 
+
+This command accepts the default flag `--help`.
+
+[[secret-group-list]]
+== secrets-mgr:secret-group:list
+
+----
+> secrets-mgr:secret-group:list [flags]
+----
+
+Displays all your secret groups. It returns the name of the secret group, and the ID.
+
+This command accepts only the default flag `--help`.
+
+[[secrets-group-modify]]
+== secrets-mgr:secret-group:modify
+
+----
+> secrets-mgr:secret-group:modify  [flags] 
+----
+
+Modifies a secret group specified in `--id`. +
+
+Besides the default flag `--help`, this command also accepts:
```

Comment:
```suggestion
In addition to the default flag `--help`, this command accepts the following flags:
```

Reply:
Or whatever the standard intro to the Flags table is.

Reply:
Same here, we always use this intro. But lets change it and I'll add this to the content audit to update